Source/Core/Core/DiscordPresence.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DiscordPresence.h"
```

Comment:
This should be a relative include (e.g. if moved to UICommon, this should be `UICommon/DiscordPresence.h`